keyboards/handwired/dactyl_manuform/6x6/blackpill_f411/rules.mk

```diff
@@ -0,0 +1,32 @@
+# MCU name
+MCU = STM32F411
+
+# Bootloader selection
+BOOTLOADER = tinyuf2
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+NKRO_ENABLE = no            # USB Nkey Rollover
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+
+# Build Options
+#   change yes to no to disable
+#
+KEYBOARD_SHARED_EP = yes
```

Comment:
```suggestion
```

Reply:
Man, this fixed my problems with keyboard detection after walking up. Big thanks :star_struck:

Reply:
done